File: SIMauricio/JAVA_TESTES/AULA02/Banco/ContaPoupanca.java
package Banco;

public class ContaPoupanca extends Conta {
    
    //Construtores da Classe Filha
    ContaPoupanca(){
    }
        
    ContaPoupanca(String agencia, String conta, float saldo){
        //Chama o Construtor da Classe Mãe
        super(agencia, conta, saldo); 
    }

    public void lancarJuros(float taxaJuros){
        if (taxaJuros >0.0f){
            this.saldo = this.saldo + (this.saldo*(taxaJuros/100));
        }
    }
}

File: SIMauricio/PYTHON_TESTES/AULA02/Banco/ContaPoupanca.py
from Conta import Conta
class ContaPoupanca(Conta):
    def __init__(self, agencia, conta, saldo):
        super().__init__(agencia, conta, saldo)

    def lancarJuros(self, taxaJuros):
        if (taxaJuros >0.0):
            self.saldo = self.saldo + (self.saldo*(taxaJuros/100))
